app/test_views/dynamic_height_title_view.rb

```diff
@@ -0,0 +1,14 @@
+class DynamicHeightTitleView < UIView
+  attr_accessor :title
+
+  def title=(t)
```

Comment:
I need some way to actually set the title FROM the test and there's no way to do that unless I repurpose the `title=` call. That method gets called if there's a title string specified as well as the `:title_view` custom class.